Userland/Libraries/LibJS/Lexer.cpp

```diff
@@ -163,7 +165,6 @@ void Lexer::consume()
                 type = "LINE SEPARATOR";
             else
                 type = "PARAGRAPH SEPARATOR";
-            dbgln("Found a line terminator: {}", type);
```

Comment:
The whole point of this block is to print something, why are you removing this?

Reply:
Whoops thought that was my debugging code